mix.exs

```diff
@@ -39,10 +37,9 @@ defmodule Mana.MixProject do
   defp deps do
     [
       {:ex_rlp, "~> 0.3.1"},
-      {:poison, "~> 4.0.1", runtime: false},
```

Comment:
What do you think about listing `Jason` as a dependency here just to be explicit?